src/System.Private.CoreLib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -380,6 +382,58 @@ public static bool StartsWith(this ReadOnlySpan<char> span, ReadOnlySpan<char> v
                     CultureInfo.CurrentCulture.CompareInfo.IsPrefix(span, value, string.GetCaseCompareOfComparisonCulture(comparisonType));
         }
 
+        [Theory]
```

Comment:
@Anipik This change got mixed up from the test somehow. I guess it was manual merge or something. Could you please fix it up?

Reply:
Actually, it is a bad change that got checked in to CoreFX. I will fix it up.